src/axom/slic/streams/SynchronizedStream.hpp

```diff
@@ -73,7 +73,15 @@ class SynchronizedStream : public LogStream
                       bool filter_duplicates);
 
   /*!
-   * \brief Dumps the messages to the console in rank-order.
+   * \brief Dumps the messages to the console in rank-order for
+   *        just the current rank
```

Comment:
I think we lose `rank-order` here.

Consider changing this to something like:
```suggestion
   * \brief Dumps the current rank's messages to the console (non-collectively)
```